src/pollution_model/run_pollution_model.py

```diff
@@ -0,0 +1,454 @@
+            raise ValueError(f"Given surface type is not valid for computing total metal load. Needed a roof, "
+                             f"but got {SurfaceType(surface_type).name}.")
+    # Define the initial and second stage metal concentrations (X_0 and X_est)
+    initial_copper_concentration = (b[0] * rainfall_ph ** b[1]) * (b[2] * antecedent_dry_days ** b[3]) * (
+            b[4] * average_rain_intensity ** b[5])
+    second_stage_copper = b[6] * rainfall_ph ** b[7]
+
+    initial_zinc_concentration = (c[0] * rainfall_ph + c[1]) * (c[2] * antecedent_dry_days ** c[3]) * (
+            c[4] * average_rain_intensity ** c[5])
+    second_stage_zinc = c[6] * rainfall_ph + c[7]
+
+    # Define Z as per experimental data
+    z = 0.75
+
+    # Define K, the wash off coefficient. TODO: find out what the k parameter should be
+    k = 1
+
+    # Initialise total copper and zinc loads as a guaranteed common factor
+    total_copper_load = initial_copper_concentration * surface_area * (1 / k)
+    total_zinc_load = initial_zinc_concentration * surface_area * (1 / k)
+
+    # Calculate total metal loads, where the method depends on if Z is less than event_duration
+    if event_duration <= z:
+        factor = (1 - math.exp(k * average_rain_intensity * event_duration))
+        total_zinc_load *= factor
+        total_copper_load *= factor
+    else:
+        factor = (1 - math.exp(k * average_rain_intensity * z))
+        bias_factor = average_rain_intensity * (event_duration - z)
+        total_zinc_load = total_zinc_load * factor + second_stage_zinc * surface_area * bias_factor
+        total_copper_load *= total_copper_load * factor + second_stage_copper * surface_area * bias_factor
+
+    return total_copper_load, total_zinc_load
+
+
+def total_metal_load_road_carpark(tss_surface):
+    """
+    Calculate the total metal load for car parks and roads from their total suspended solids.
+
+    Parameters
+    ----------
+    tss_surface: float
+        total suspended solids of this surface
+
+    Returns
+    -------
+    float, float
+       Returns the total copper and zinc loads for this surface
+    """
+    # Define constants
+    proportionality_constant_cu = 0.441
+    proportionality_constant_zn = 1.96
+    # Return total copper load, total zinc load
+    return tss_surface * proportionality_constant_cu, tss_surface * proportionality_constant_zn
+
+
+def dissolved_metal_load(total_copper_load, total_zinc_load, surface_type):
+    """
+        Calculate the dissolved metal load for all surfaces from their total suspended solids.
+
+        Parameters
+        ----------
+        total_copper_load: float
+            total copper load for the surface
+        total_zinc_load: float
+            total zinc load for the surface
+        surface_type: int
+            The type of surface that we are calculating this for
+
+        Returns
+        -------
+        float, float

```

Comment:
```suggestion
        tuple[float, float]

```

Reply:
Repeat this in some other areas too, and document what each of the floats mean. It may be worth using a `namedtuple` or a `dataclass` since the order of those floats is so important